zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -96,7 +90,9 @@ fn create_close_and_report_market<T: Config>(
     options: MarketType,
     outcome: OutcomeReport,
 ) -> Result<(T::AccountId, MarketIdOf<T>), &'static str> {
-    let (caller, market_id) = create_market_common::<T>(permission, options, ScoringRule::CPMM)?;
+    let period = MarketPeriod::Timestamp(T::MinSubsidyPeriod::get()..T::MaxSubsidyPeriod::get());
```

Comment:
You don't have to fix this, but I still don't understand what `MaxSubsidyPeriod` is doing here.

Reply:
I can change it to standard values here, if you'd like!

Reply:
19aaf9bbdce77af69975a0dd82f06abee713a0e6